coverage/.resultset.json

```diff
@@ -1,18 +1,34 @@
 {
   "RSpec": {
     "coverage": {
```

Comment:
You need to clear the cache as i showed Viktor yesterday, so the coverage is not pushed up everytime